Repository/FormSubmissionRepository.php

```diff
@@ -40,4 +40,10 @@ public function save(FormSubmission $formSubmission): void
         $this->_em->persist($formSubmission);
         $this->_em->flush();
     }
+
+    public function remove(FormSubmission $formSubmission): void
+    {
+        $this->_em->remove($formSubmission);
+        $this->_em->flush();
```

Comment:
I'll check but why will you flush all pending updates here?

Reply:
OK I'll create a separate flush function and only call it once.